pkgs/tools/security/gnupg/minimal.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, gnupg }:
+
+let 
+  p = gnupg.override {
+    pcsclite = null;
+    sqlite = null;
+    pinentry = null;
+    guiSupport = false;
+    adns = null;
+    gnutls = null;
+    libusb = null;
+    openldap = null;
+    readline = null;
+    zlib = null;
+    bzip2 = null;
+  };
+in
+  stdenv.mkDerivation rec {
```

Comment:
can't you use `overrideAttrs` together with `override` here?

Or merge the whole `postInstall` cleanup into the main gnupg derivation, behind a `enableMinimal` flag defaulting to false?

Reply:
But that way the build itself becomes fat again, since one at least has to fetch all that stuff for building a normal `gnupg`?

Reply:
I meant, moving the `postInstall` cleanup stuff into the gnupg derivation itself, but still calling

`gnupg.override { enableMinimal = true; pcsclite = null; sqlite = null; … }`